scripts/rollup/mangle.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// these are added as they are handled by rollup in some cases
+const propertyMangleWhitelist = [
+  // React
```

Comment:
Can you explain more about this change and why it is safe? I would prefer that we don't alter the build significantly in this PR, and pursue more aggressive strategies separately.